erpnext/hr/doctype/expense_claim/expense_claim.py

```diff
@@ -360,3 +362,47 @@ def get_expense_claim(
 	)
 
 	return expense_claim
+
+# API to create expense claim
+@frappe.whitelist()
+def create_expense_claim(user, expense_date, expense_claim_type, description, amount):
+	"""
+	Creating Expense Claim document as user requested.
+	Args:
+		user: string -> user_id of an Employee
+		expense_date: date -> Date when the expense were made
+		expense_claim_type: string -> Type of expense claim i.e. Calls, Food, Travel, Others, Vehicle, Medical
+		description: string -> Complete detail of the expense claim like 'Why the user is applying, is that claim covers in company policy etc.'
+		amount: currency -> Total amount that user wants to claim
+	Returns:
+		Nothing
+	"""
+	# Fetching employee document based on user_id.
+	employee = frappe.get_doc("Employee", {"user_id": user})
+	
+	expense_approver = ""
+	# Based on Department it fetches the expense_approver list.
```

Comment:
Fetch list of expense approvers based on the department of the employee claiming the expense